src/odemis/acq/test/stream_test.py

```diff
@@ -3383,12 +3386,16 @@ def on_im(im):
             self.fail("Failed to find a second point in AR")
 
         e.wait()
+
+        time.sleep(5)  # TODO e.wait() seems to be not to be enough...why?? And why so long?
```

Comment:
That's a good question indeed. One possibility is that .image is updated more than twice (ie, `on_im()` is called more than twice). Maybe the first two times, it's updated with the original image, and then later with the new image. You could check relatively easily by adding a log message in `on_im()`.
I've tried on the current version of Odemis (ie, without polarymetry update), and it also happens.
Short tests show that every image update is notified twice.